src/components/shared/Picture.tsx

```diff
@@ -28,8 +33,18 @@ const Picture = ({ source, uri, style }: PictureProps) => {
 
   return (
     <Fragment>
-      <Image style={getStyle()} source={getSourceForImage()} onLoadEnd={() => setLoader(false)} />
-      {loader && <ActivityIndicator style={styles.loading} animating={loader} size="large" />}
+      <Image
+        style={getStyle()}
+        source={getSourceForImage()}
+        onLoadEnd={() => setLoader(false)}
+      />
+      {loader && (
+        <ActivityIndicator
+          style={styles.loading}
+          animating={loader}
+          size="large"
+        />
```

Comment:
We should use NativeBase components - that way we don't need to think about dark-light mode since it's already probably handled by components 
https://docs.nativebase.io/spinner